src/Uno.UI.RemoteControl.Host/Uno.UI.RemoteControl.Host.csproj

```diff
@@ -53,5 +53,6 @@
 	</Target>
 
 	<Target Name="GetTargetPath" />
+	<Target Name="GetCopyToPublishDirectoryItems" />
```

Comment:
@jeromelaban The build fails for me without this (on master branch too). If this change doesn't look correct, let me know how I can resolve the build failures I see locally.

Reply:
That's fine, I'm not too sure of what causes these to happen regularly.